src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
i see now:  graylog prints host in its log entry, so `Monitor::update_log_clients()` also reads from conf.host and pass the `host` read from conf to graylog.